refractr/base.py

```diff
@@ -35,10 +38,24 @@ def __str__(self):
     @property
     def src(self):
         if self.srcs:
-            assert isinstance(self.srcs, tuple) and len(self.srcs) > 0
+            assert isinstance(self.srcs, tuple) and len(self.srcs) > 0, type(self.srcs)
             return self.srcs[0]
         return None
 
+    @property
+    def srcs(self):
+        if self.wildcard_file:
```

Comment:
if we wanted to have a domain with the word 'wildcard' that would be possible so long as we either include it in the wildcard file, or create the refract without the wildcard file?  Is that edge case worth documenting somewhere?

Reply:
you can have a domain with prefix of wildcard as long as you dont add `wildcard_file` in the yaml spec; if you do add that I think we can assume you know what you are doing.